src/main/features/ccj/index.ts

```diff
@@ -12,7 +13,11 @@ import { CCJGuard } from 'ccj/guards/ccjGuard'
 
 function requestHandler (): express.RequestHandler {
   function accessDeniedCallback (req: express.Request, res: express.Response): void {
-    res.redirect(`${config.get('idam.authentication-web.url')}/login?continue-url=${buildURL(req, AppPaths.receiver.uri)}`)
+    const clientId = config.get<string>('oauth.clientId')
+    const continueUrl = `${buildURL(req, AppPaths.receiver.uri.substring(1))}`
```

Comment:
rename to redirectUri

Reply:
also remove the template string as you aren't using anymore, just calling a function